n_actions: 3 # We have 3 actions, Forwards,TurnLeft,TurnRight
n_observations: 5 # We have 6 different observations

speed_step: 1.0 # Time to wait in the reset phases

linear_forward_speed: 0.5 # Spawned for ging fowards
linear_turn_speed: 0.1 # Lienare speed when turning
angular_speed: 0.8 # Angular speed when turning Left or Right
init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode
    
new_ranges: 5 # How many laser readings we jump in each observation reading, the bigger the less laser resolution
min_range: 0.2 # Minimum meters below wich we consider we have crashed
max_laser_value: 6 # Value considered Ok, no wall
min_laser_value: 0 # Value considered there is an obstacle or crashed

model_ckpt: '/home/eldar/python3_ws/src/turtle2_openai_ros_example/src/checkpoints/dqn-episode-756-step-32000.pt'
    
number_of_sectors: 3 # How many sectors we have

middle_range: 1.0 # Minimum meters below wich we consider we have crashed
danger_laser_value: 2 # Value considered Ok, no wall
middle_laser_value: 1 # Middle value
safe_laser_value: 0 # Value considered there is an obstacle or crashed
    
forwards_reward: 5 # Points Given to go forwards
turn_reward: 4 # Points Given to turn as action
end_episode_points: 200 # Points given when ending an episode
